### PR TITLE
Various small bugfixes

### PR DESCRIPTION
**Pull request by zelzmiy**
Note: Keep branch active after merge, use as branch for minor, unrelated bug fixes.

- Fix GameLights being enabled erroneously
-  fix Issue #39 (Global Toggle not working)
- Fix sending player to casino while it's disabled

### CODE REVIEW

**Comment by zelzmiy** [comment]
Latest commits Fix #33

File: src/CultOfJakito.UltraTelephone2/Chaos/Effects/GamerLights.cs [hunk 1]
```diff
@@ -68,7 +68,7 @@ private void SlowTickGetLights()
 
         private void LateUpdate()
         {
-            if (!effectActive)
+            if (!effectActive && !s_enabled.Value)
```
Comment (Hydraxous) [inline src/CultOfJakito.UltraTelephone2/Chaos/Effects/GamerLights.cs:71]: || operator

File: src/CultOfJakito.UltraTelephone2/Chaos/Effects/MovingWindow/WindowDanceEffect.cs [hunk 2]
```diff
@@ -38,6 +38,7 @@ private struct Rect //sucks, but needed for getwindowrect
 
     public override void BeginEffect(UniRandom random)
     {
+        s_enabled.OnValueChanged += DisableEffect;
```
Comment (Hydraxous) [inline src/CultOfJakito.UltraTelephone2/Chaos/Effects/MovingWindow/WindowDanceEffect.cs:41]: Make sure to unsubscribe this in OnDestroy

File: src/CultOfJakito.UltraTelephone2/Chaos/Effects/AltMarksmanOnly.cs [hunk 3]
```diff
@@ -11,9 +11,18 @@ public class AltMarksmanOnly : ChaosEffect
 
         public override void BeginEffect(UniRandom random)
         {
+            s_enabled.OnValueChanged += DisableForcedLoadout;
```
Comment (Hydraxous) [inline src/CultOfJakito.UltraTelephone2/Chaos/Effects/AltMarksmanOnly.cs:14]: Need to unsubscribe this in OnDestroy

**Review by Hydraxous — CHANGES_REQUESTED** [review]

**Review by Hydraxous — APPROVED** [review]

**Pull request merged**